docs/tutorials/otio-env-variables.md

```diff
@@ -18,11 +18,6 @@ OTIO_DEFAULT_MEDIA_LINKER
    The name of the default media linker to use after reading a file, if `""` then no
    media linker is automatically invoked.
 
-OTIO_DISABLE_PKG_RESOURCE_PLUGINS
-   By default, OTIO will use the `pkg_resource` entry_points mechanism to discover plugins
-   that have been installed into the current python environment. `pkg_resources`, however, can
-   be slow in certain cases, so for users who wish to disable this behavior, this variable can be set to 1.
-
```

Comment:
Let's keep the environment variable but update the name of it, just in case anyone in the future has problems with this feature (even via the new `importlib` path you're adding.

Reply:
Yes, please. In some contexts where we know that plugins aren't in use, it will be good to have an option to turn this off & avoid the overhead of looking for them. A toggle is also hypothetically helpful in some security scenarios where you want to limit runtime discovery of untrusted code.